src/components/PopupWithForm.js

```diff
@@ -7,44 +7,42 @@ export default class PopupWithForm extends Popup {
     this._popupForm = this._popupElement.querySelector(".modal__form");
     this._inputs = this._popupElement.querySelectorAll(".modal__input");
     this._submitButton = this._popupElement.querySelector(
-      ".modal__submit-button"
+      ".modal__button"
     );
+    this._submitButtonText = this._submitButton.textContent;
   }
 
   close() {
-    this._popupForm.reset();
     super.close();
+    this._popupForm.reset();
   }
 
   _getInputValues() {
     const _inputs = Array.from(
-      this.popupForm.querySelectorAll(".modal__input")
+      this._popupForm.querySelectorAll(".modal__input")
     );
     this._inputValues = {};

```

Comment:
Needs correction: You should not store variables in class instance `this` until it's needed. If the variables are not used in other methods, then it’s needed to declare them as local constants.

```js
const inputValues = ...
```

Reply:
I am confused by the review and approval process through pull requests. My last attempt still refers to this code even though my changes to my code reflect the fixes. I've tried making a new pull request and submitting it but this one keeps being referred.

Reply:
You should make the changes in the same local branch and send the changes to that remote branch. The changes will be automatically displayed in your pull request.